File: Python-Basics/walking.py
steps_goal = 10000

sum_steps = 0
while sum_steps <= steps_goal:
    command = input()
    if command == "Going home":
        steps_home = int(input())
        sum_steps += steps_home
        break

    current_steps = int(command)
    sum_steps += current_steps


diff = abs(sum_steps - steps_goal)
if sum_steps >= steps_goal:
    print("Goal reached! Good job!")
    print(f"{diff} steps over the goal!")
else:
    print(f"{diff} more steps to reach goal.")

File: JS-Basics/While Loop Exercise/walking.js
function walking(input){
    let index = 0;
    let goal = 10000;
    let diff = 0;
    let command = input[index];
    index++
    let totalSteps = 0;
    let isSuccessful = true;
    while(totalSteps <= goal){
        if(command === "Going home"){
            bonus_steps = Number(input[index]);
            totalSteps += bonus_steps;
            isSuccessful = false;
            break;
        }
        current_steps = Number(command);
        totalSteps += current_steps;

        command = input[index];
        index++
    }
    diff = Math.abs(totalSteps - goal);
    if(isSuccessful === true){
        console.log("Goal reached! Good job!")
        console.log(`${diff} steps over the goal!`)
    }
    else{
        console.log(`${diff} more steps to reach goal.`)
    }

}
walking(["1500",
"3000",
"250",
"1548",
"2000",
"Going home",
"2000"])

;
